idea/testData/inspectionsLocal/conventionNameCalls/replaceCallWithBinaryOperator/equalsFloat.kt

```diff
@@ -0,0 +1,5 @@
+// PROBLEM: none
+
+fun test(a: Any, b: Any) =
```

Comment:
Please add also test with explicitly given `Float` or `Double` types, like this: `fun test(a: Double, b: Double) = ...`, and another test with `Double` or `Float` literals, like `fun checkPi(x: Double) = x.equals(3.14) // or 3.14.equals(x)`

Reply:
I added tests.